integreat_cms/release_notes/current/unreleased/3131.yml

```diff
@@ -0,0 +1,2 @@
+en: Make access number box of statistics collapsible
+de: Mache Zugriffsnummernfeld der Statistik zusammenklappbar
```

Comment:
Thank you for adding this. I must have also missed this during my review of the other PR. I'm sorry! :see_no_evil: